test/e2e/workflow_test.go

```diff
@@ -213,34 +213,34 @@ var _ = Describe("Kogito Serverless Operator", Ordered, func() {
 			By("creating an instance of the Kogito Serverless Workflow in DevMode")
 			EventuallyWithOffset(1, func() error {
 				cmd := exec.Command("kubectl", "apply", "-f", filepath.Join(projectDir,
-					test.GetServerlessWorkflowE2eTest()), "-n", namespace)
+					test.GetServerlessWorkflowE2eOrderProcessingFolder()), "-n", namespace)
```

Comment:
Please change the name of the test (line #211) as you test here `order processing` and not `greeting`